src/config.h

```diff
@@ -556,6 +542,23 @@ class _Config_t
     /* ---- Advanced - Audio ----*/
     subsection advanced_audio{this, "advanced-audio", "Audio"};
 
+    opt<bool> audio_enable{this, defaults(true), {}, Scope::User,
+        "audio-enable", "Enable", nullptr,
+        config_audio_set};
+
+#ifndef _WIN32
+    opt_enum<int> audio_channels{this,
+        {
+            {1, "mono", "Mono", nullptr},
+            {2, "stereo", "Stereo", nullptr},
+        },
+        defaults(g_audioDefaults.channels), {}, Scope::User,
+        "audio-channels", "Channels", nullptr,
+        config_audio_set};
+#else // #ifndef _WIN32
+    static constexpr int audio_channels = 2;
```

Comment:
Just a note: while default requests the 2, it's possible when the 5.1 or 7.1 output has been configured, then it will use 6 or 8 channels actually (such number of channels will be reported via Obtained, and will be used de-facto).